Vis items.location i bold etter callnumber. Vi har ikke autoriserte verdier for location code (LOC),
så vi vil vise koden, drop å let etter en ikke-eksisterende label.

diff --git a/koha-tmpl/intranet-tmpl/prog/en/modules/circ/view_holdsqueue.tt b/koha-tmpl/intranet-tmpl/prog/en/modules/circ/view_holdsqueue.tt
index 67f3fc4..761bd2c 100644
--- a/koha-tmpl/intranet-tmpl/prog/en/modules/circ/view_holdsqueue.tt
+++ b/koha-tmpl/intranet-tmpl/prog/en/modules/circ/view_holdsqueue.tt
@@ -168,7 +168,7 @@ $(document).ready(function() {
             </td>
             <td class="hq-collection">[% AuthorisedValues.GetByCode( 'CCODE', itemsloo.ccode ) %]</td>
             <td class="hq-itemtype">[% ItemTypes.GetDescription( itemsloo.itype ) %]</td>
-            <td class="hq-callnumber">[% IF ( itemsloo.location ) %]<em>[% AuthorisedValues.GetByCode( 'LOC', itemsloo.location ) %]</em> [% END %][% itemsloo.itemcallnumber %]</td>
+            <td class="hq-callnumber">[% itemsloo.itemcallnumber %] [% IF ( itemsloo.location ) %]<strong>[% itemsloo.location %]</strong>[% END %]</td>
             <td class="hq-copynumber">[% itemsloo.copynumber %]</td>
             <td class="hq-enumchron">[% itemsloo.enumchron %]</td>
             <td class="hq-barcode">
